osbenchmark/resources/default-test-procedures.json.j2

```diff
@@ -0,0 +1,41 @@
+{ 
+  "name": "append-no-conflicts",
+  "description": "Indexes the whole document corpus using OpenSearch default settings. We only adjust the number of replicas as we benchmark a single node cluster and Benchmark will only start the benchmark if the cluster turns green. Document ids are unique so all index operations are append only.",
+  "default": true,
+  "schedule": [
+    {
+      "operation": "delete-index"
+    },{% raw %}
+    {
+      "operation": {
+        "operation-type": "create-index",
+        "settings": {{index_settings | default({}) | tojson}}
+      }
+    },{% endraw %}
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "index": {{ indices | map(attribute='name') | list | join(',') | tojson }},{% raw %}
+        "request-params": {
+          "wait_for_status": "{{cluster_health | default('green')}}",
+          "wait_for_no_relocating_shards": "true"
+        },
+        "retry-until-success": true
+      }
+    },{% endraw -%}
+    {
+      "operation": "index-append",
+      {% raw -%} "clients": {{bulk_indexing_clients | default(8)}},{% endraw -%}
+      {% raw -%}"ignore-response-error-level": "{{error_level | default('non-fatal')}}"{% endraw -%}
+    },
+    {
+      "name": "refresh-after-index",
+      "operation": "refresh"
+    }, 
+    {
+      "name": "search-after-index",
```

Comment:
We should still include basic workload parameters such as `warmup-iterations`, `iterations`, `target-throughput`, and `clients. See https://github.com/rishabh6788/opensearch-benchmark-workloads/blob/main/http_logs/test_procedures/default.json#L60-L71 for reference. Doesn't have to be identical to this since we're trying to consolidate this format, as seen in this issue https://github.com/opensearch-project/opensearch-benchmark-workloads/issues/120